extensions/wikia/WikiaHomePage/WikiaHomePage.setup.php

```diff
@@ -44,5 +43,4 @@
 $wgHooks['ArticleCommentCheck'][] = 'WikiaHomePageController::onArticleCommentCheck';
 $wgHooks['AfterGlobalHeader'][] = 'WikiaHomePageController::onAfterGlobalHeader';
 $wgHooks['BeforePageDisplay'][] = 'WikiaHomePageController::onBeforePageDisplay';
-$wgHooks['AfterOasisSettingsInitialized'][] = 'WikiaHomePageController::onAfterOasisSettingsInitialized';

```

Comment:
Let's kill this hook, there seems to be nothing else using it

Reply:
Removed in c685ced